build.rs

```diff
@@ -131,6 +130,37 @@ fn prepare_tensorflow_library() {
                 .arg("-f")
                 .arg("tensorflow/lite/tools/make/Makefile");
 
+            for &make_var in &[
+                "TARGET",
+                "TARGET_ARCH",
+                "TARGET_TOOLCHAIN_PREFIX",
+                "EXTRA_CFLAGS",
+            ] {
```

Comment:
is it possible to list pairs of env variables and fallback values?
```rust
            for &make_var in &[
                ("TARGET", Some(target)),
                ("TARGET_ARCH", Some(arch)),
                ("TARGET_TOOLCHAIN_PREFIX", None),
                ("EXTRA_CFLAGS", None),
            ] 
```

Reply:
It is! Pushed a branch with that change.